relay-common/Cargo.toml

```diff
@@ -12,6 +12,7 @@ publish = false
 [dependencies]
 backoff = "0.1.5"
 cadence = "0.17.1"
+chrono = { version = "0.4.7", optional = true }
```

Comment:
Why's chrono optional, isn't that feature used all the time anyway?

Reply:
Didn't bother looking, but since it wasn't there and I only needed it in the `processing` feature, it was straight-forward.